zerver/models.py

```diff
@@ -3626,6 +3652,17 @@ def get_old_unclaimed_attachments(
 
 
 class Subscription(models.Model):
+    """Keeps track of which users are part of the
+    audience for a given Recipient object.
+
+    For private and group private message Recipient objects, only the
```

Comment:
Is there a Subscription object involved in 1:1 PMs? IIUC there isn't -- so this can just say "For group private message Recipient objects, …".

(If 1:1 PMs do indeed involve these, then that rewording isn't it. But this wording sounds like it's saying "private message" to mean specifically 1:1 PMs, which I'd like to avoid as discussed at https://github.com/zulip/zulip/pull/22919#discussion_r968287102 .)

Reply:
We do create such a `Subscription` object as part of creating a user. It's mostly not used -- in that code often just looks at the `type_id` to find the user in question -- but it'd be correct for such code to do a `Subscription` table lookup instead.